packages/controls/src/widget_bool.ts

```diff
@@ -301,13 +301,14 @@ export class ValidView extends DescriptionView {
   /**
    * Called when view is rendered.
    */
-  render(): void {
+  render() {
```

Comment:
Do you need a rebase?

Reply:
xref https://github.com/jupyter-widgets/ipywidgets/pull/2808

Reply:
Well, the merge would have taken care of it, presumably, but I rebased nonetheless.

Reply:
Well, except it wouldn't take care of an explicit change... I'll fix it in a new commit.

Reply:
This should be resolved now.